src/analyze/analyze.c

```diff
@@ -2447,6 +2470,10 @@ static int parse_argv(int argc, char *argv[]) {
                 return log_error_errno(SYNTHETIC_ERRNO(EINVAL),
                                        "Option --user is not supported for cat-config right now.");
 
+        if (arg_json_path && !streq_ptr(argv[optind], "security"))
+                return log_error_errno(SYNTHETIC_ERRNO(EINVAL),
+                                       "Option --json-path= is only supported for security right now.");
```

Comment:
"right now"? unlikely to be used for anything else ever , hence drop these two words